open_academy/views/course_views.xml

```diff
@@ -4,11 +4,12 @@
         <field name="name">Courses</field>
         <field name="res_model">course</field>
         <field name="view_mode">tree,form</field>
+        <field name="context">{"search_default_my_courses": 1}</field>
     </record>
 
-    <menuitem id="open_academy_root_menu" name="Courses" sequence="1"/>
+    <menuitem id="open_academy_root_menu" name="Courses" sequence="2"/>
```

Comment:
I updated because the exercise says the Dashboard needs to be the default Menu when the `open_academy` module is selected

> Define a dashboard containing the graph view you created, the sessions calendar view and a list view of the courses (switchable to a form view). This dashboard should be available through a menuitem in the menu, **and automatically displayed in the web client when the OpenAcademy main menu is selected.**

Reply:
Ok, thanks